tests/specs/constructor.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+var test = require('tape')
+
+var dbFactory = require('../utils/db')
+var Store = require('../../')
+
+test('constructs a store object', function (t) {
+  t.plan(2)
+
+  var db = dbFactory()
+  var store = new Store(db)
+
+  t.is(typeof store, 'object', 'is object')
+  t.is(store.db, db, 'exposes db')
```

Comment:
@gr2m how should I test for the db being created by the constructor without a db utility? 

Should I keep the db utility to compare the exposed db created by the dbName?